openerp/addons/base/security/ir.model.access.csv

```diff
@@ -28,7 +28,7 @@
 "access_ir_sequence_group_system","ir_sequence group_system","model_ir_sequence","group_system",1,1,1,1
 "access_ir_sequence_date_range_group_user","ir_sequence_date_range group_user","model_ir_sequence_date_range","group_user",1,0,0,0
 "access_ir_sequence_date_range_group_system","ir_sequence_date_range group_system","model_ir_sequence_date_range","group_system",1,1,1,1
-"access_ir_translation_all","ir_translation all","model_ir_translation",,1,1,1,1
+"access_ir_translation_all","ir_translation all","model_ir_translation",,1,0,0,0
```

Comment:
Hmm, I guess the easiest fix would be to modify the access line for `group_system` to grant it to `group_user` instead. The sudo() would not be actually necessary in the various ORM overrides in that case, would it? (perhaps for odd cases where external users have write access to some translatable fields..)
Moreover, not using sudo mode would mean that the real access rights still apply, so if someone wants to really lock down all translations for some reason, they could still do it.